Makefile

```diff
@@ -161,7 +155,7 @@ realclean: clean
 	fi
 
 shell: work
-	cd $(DEST) && $(SHELL) -i
+	$(SHELL) -i
 
 images: image-controller-manager image-flex-volume-driver image-provisioner image-csi-plugin image-k8s-keystone-auth
```

Comment:
The image builds below should also use $(BIN_DIR)

Reply:
Oh GREAT catch!  I'm trying an experiment right now to see if that BIN_DIR change is screwing up the build, but new to the OpenLab stuff and haven't quite untangled yet.